deployment/local-compose/README.md

```diff
@@ -0,0 +1,159 @@
+# Run a local OpenWPM crawl using Docker Compose
+
+Documentation and scripts to launch an OpenWPM crawl using Docker Compose locally.
```

Comment:
We should have a more descriptive summary here of how this compares to the other local option, particularly for someone that isn't familiar with Docker Compose.

We should also add a description to the root directory readme as we've done for the other deployment options.

Reply:
I added some more prose. I hope my bias isn't overshadowing useful information.